app/models/Date.php

```diff
@@ -1,44 +1,44 @@
 <?php
     class Date
```

Comment:
This class'll be really good to add unit tests too, all the complex business logic being validated would be hugely confidence boosting